library/scan_facts.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
yeah, pick something with a name that really has meaning only in a test context.